core/client/hdfs/src/main/java/alluxio/hadoop/HadoopConfigurationUtils.java

```diff
@@ -53,9 +53,7 @@ public static InstancedConfiguration mergeHadoopConfiguration(
         alluxioConfProperties.put(propertyName, entry.getValue());
       }
     }
-    LOG.info("Loading Alluxio properties from Hadoop configuration: {}", alluxioConfProperties);
```

Comment:
we shall not `LOG.info` inside the util, moved to `AbstractFileSystem.initialize`